awx/main/tests/unit/scheduler/test_dag_workflow.py

```diff
@@ -19,7 +19,15 @@ def __init__(self, id=None, job=None, do_not_run=False, unified_job_template=Non
         self.job = job
         self.do_not_run = do_not_run
         self.unified_job_template = unified_job_template
+        self.all_parents_must_converge = False
 
+class ConvergenceNode(object):
+    def __init__(self, id=None, job=None, do_not_run=False, unified_job_template=None):
+        self.id = id if id is not None else uuid.uuid4()
+        self.job = job
+        self.do_not_run = do_not_run
+        self.unified_job_template = unified_job_template
+        self.all_parents_must_converge = True
```

Comment:
I think this never got used?

Reply:
you are completely right, I'll remove as I fix the rest of the test statements.